core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/SixteenByNineFrameLayout.kt

```diff
@@ -16,11 +17,34 @@ open class SixteenByNineFrameLayout: FrameLayout {
     constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
     constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(context, attrs, defStyleAttr)
 
+    private var customOnTouchListener: OnTouchListener? = null
+
     override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
         if (layoutParams.height == ViewGroup.LayoutParams.WRAP_CONTENT) {
             val sixteenNineHeight = View.MeasureSpec.makeMeasureSpec(View.MeasureSpec.getSize(widthMeasureSpec) * 9 / 16, View.MeasureSpec.EXACTLY)
             super.onMeasure(widthMeasureSpec, sixteenNineHeight)
         } else
             super.onMeasure(widthMeasureSpec, heightMeasureSpec)
     }
+
+    /**
+     * This method is usually used to intercept events.
+     * @return Return true to steal motion events from the children
+     * and have them dispatched to this ViewGroup through onTouchEvent()
+     */
+    override fun onInterceptTouchEvent(ev: MotionEvent): Boolean {
```

Comment:
I think the description of why we are overriding this method got lost?

Reply:
lol, I thought you wanted it that way: https://github.com/PierfrancescoSoffritti/android-youtube-player/pull/521#discussion_r379856256

Will bring the description back.

Reply:
Ah sorry, with that comment I meant "rephrase the first line" :)